framework/src/transfers/MultiAppMeshFunctionTransfer.C

```diff
@@ -265,88 +255,66 @@ MultiAppMeshFunctionTransfer::transferVariable(unsigned int i)
   }
 
-  // Send points to other processors.
-  std::vector<std::vector<Real>> incoming_evals(n_processors());
-  std::vector<std::vector<unsigned int>> incoming_app_ids(n_processors());
-  _send_points[i].resize(n_processors());
-  for (processor_id_type i_proc = 0; i_proc < n_processors(); ++i_proc)
-  {
-    if (i_proc == processor_id())
-      continue;
-
-    _communicator.send(i_proc, outgoing_points[i_proc], _send_points[i][i_proc]);
-  }
-
-  // Receive points from other processors, evaluate mesh functions at those
-  // points, and send the values back.
-  _send_evals[i].resize(n_processors());
-  _send_ids[i].resize(n_processors());
-
-  // Create these here so that they live the entire life of this function
-  // and are NOT reused per processor.
-  std::vector<std::vector<Real>> processor_outgoing_evals(n_processors());
-
-  for (processor_id_type i_proc = 0; i_proc < n_processors(); ++i_proc)
-  {
-    std::vector<Point> incoming_points;
-    if (i_proc == processor_id())
-      incoming_points = outgoing_points[i_proc];
-    else
-      _communicator.receive(i_proc, incoming_points);
-
-    std::vector<Real> & outgoing_evals = processor_outgoing_evals[i_proc];
-    outgoing_evals.resize(incoming_points.size(), OutOfMeshValue);
-
-    std::vector<unsigned int> outgoing_ids(incoming_points.size(), -1); // -1 = largest unsigned int
-    for (unsigned int i_pt = 0; i_pt < incoming_points.size(); ++i_pt)
-    {
-      Point pt = incoming_points[i_pt];
-
-      // Loop until we've found the lowest-ranked app that actually contains
-      // the quadrature point.
-      for (unsigned int i_from = 0;
-           i_from < _from_problems.size() && outgoing_evals[i_pt] == OutOfMeshValue;
-           ++i_from)
-      {
-        if (local_bboxes[i_from].contains_point(pt))
-        {
-          outgoing_evals[i_pt] = (*local_meshfuns[i_from])(pt - _from_positions[i_from]);
-          if (_current_direction == FROM_MULTIAPP)
-            outgoing_ids[i_pt] = _local2global_map[i_from];
-        }
-      }
-    }
-
-    if (i_proc == processor_id())
-    {
-      incoming_evals[i_proc] = outgoing_evals;
-      if (_current_direction == FROM_MULTIAPP)
-        incoming_app_ids[i_proc] = outgoing_ids;
-    }
-    else
-    {
-      _communicator.send(i_proc, outgoing_evals, _send_evals[i][i_proc]);
-      if (_current_direction == FROM_MULTIAPP)
-        _communicator.send(i_proc, outgoing_ids, _send_ids[i][i_proc]);
-    }
-  }
-
   /**
    * Gather all of the evaluations, pick out the best ones for each point, and
    * apply them to the solution vector.  When we are transferring from
    * multiapps, there may be multiple overlapping apps for a particular point.
    * In that case, we'll try to use the value from the app with the lowest id.
    */
 
-  for (processor_id_type i_proc = 0; i_proc < n_processors(); ++i_proc)
-  {
-    if (i_proc == processor_id())
-      continue;
+  // Fill values and app ids for incomming points
```

Comment:
`incomming` -> `incoming`

Reply:
Should just be able to do find and replace in this file

Reply:
I remembered I did. My head did not work well today

Reply:
Hopefully, I got it right this time, haha